htsinfer/get_read_orientation.py

```diff
@@ -446,7 +446,7 @@ def process_single(
                                 StatesOrientation.stranded_forward
                             )
 
-        except OSError as exc:
+        except (OSError, ValueError) as exc:
```

Comment:
Can you please explain under what conditions a `ValueError` is raised? I mean I'm sure that it can happen with all that code in the loop, especially if the file is not a valid SAM file. But it might make more sense to catch the errors separately and give a more informative error message, becaue `Failed to open SAM file` is probably not quite appropriate for `ValueError`. Same comment applies to the change below.

Reply:
So if the file is empty? Or if the records are corrupt / not valid SAM? Either way, would be good to give a more accurate/informate error message, like `Input file empty` or `Not a valid SAM file`.